Solutions/Darktrace/Analytic Rules/CreateIncidentFromAIAnalystIncident.yaml

```diff
@@ -8,50 +8,58 @@ requiredDataConnectors:
-  | project-rename EventEndTime = endTime_s
-  | project-rename NetworkRuleName=title_s
-  | project-rename CurrentGroup=externalId_g //externalId is the Current Group ID from Darktrace 
-  | project-rename ThreatCategory=dtProduct_s
-  | extend ThreatRiskLevel=score_d //This is the event score, which is different from the GroupScore
-  | project-rename SrcHostname=hostname_s
-  | project-rename DarktraceLink=url_s
-  | project-rename Summary=summary_s
-  | project-rename GroupScore=groupScore_d
-  | project-rename GroupCategory=groupCategory_s
-  | project-rename SrcDeviceName=bestDeviceName_s
+  darktrace_model_alerts_CL 
+  | where dtProduct_s =="AI Analyst"
+  | project-rename  EventStartTime=startTime_s, EventEndTime = endTime_s, NetworkRuleName=title_s, DtCurrentGroup=externalId_g, ThreatCategory=dtProduct_s, ThreatRiskLevel=score_d, SrcHostname=hostname_s, SrcIpAddr=deviceIP_s, DtURL=url_s, DtSummary=summary_s, DtGroupScore=groupScore_d, DtGroupCategory=groupCategory_s, DtSrcDeviceName=bestDeviceName_s, DtIndentifier=identifier_s, ActivityID=activityId_s, DtGroupingID=groupingId_s, DtGroupByActivity=groupByActivity_b, DtSummaryFirstSentence=summaryFirstSentence_s, DtNewEvent=newEvent_b, DtCGLegacy=currentGroup_s, DtGroupPreviousGroups=groupPreviousGroups_s, DtTime=time_s, DtSeverity=Severity, DtLongitude=longitude_d, DtLatitude=latitude_d  
+  | extend EventVendor = "Darktrace", EventProduct = "Darktrace DETECT", DtSentinelCategory=DtGroupCategory
+  | extend DtSentinelCategory = case (DtGroupCategory == "compliance", "Low", 
+                                      DtGroupCategory == "suspicious", "Medium",
+                                      "High") //compliance -> low, suspcious -> medium, critical -> high
 eventGroupingSettings:
   aggregationKind: AlertPerResult
 entityMappings: 
  - entityType: Host
    fieldMappings:
     - identifier: "HostName"
-      columnName: "SrcDeviceName"
+      columnName: "SrcHostname"
+ - entityType: IP
+   fieldMappings:
+    - identifier: Address
+      columnName: SrcIpAddr
 customDetails:
   EventStartTime: EventStartTime
-  Title: NetworkRuleName
-  CurrentGroupID: CurrentGroup
-  EventScore: ThreatRiskLevel
-  IncidentSummary: Summary
-  DarktraceLink: DarktraceLink
   EventEndTime: EventEndTime
-  GroupScore: GroupScore
-  GroupCategory: GroupCategory
-  SrcDeviceName: SrcDeviceName
+  NetworkRuleName: NetworkRuleName
+  DtCurrentGroup: DtCurrentGroup
+  ThreatRiskLevel: ThreatRiskLevel
+  DtSummary: DtSummary
+  DtGroupScore: DtGroupScore
+  DtGroupCategory: DtGroupCategory
+  DtSentinelCategory: DtSentinelCategory
+  DtSrcDeviceName: DtSrcDeviceName
+  DtNewEvent: DtNewEvent
+  DtSeverity: DtSeverity
 alertDetailsOverride: 
-  alertDisplayNameFormat: '{{NetworkRuleName}}'
-  alertDescriptionFormat: '{{Summary}}'
+  alertDisplayNameFormat: 'Darktrace: {{ThreatRiskLevel}} - {{NetworkRuleName}}'
+  alertDescriptionFormat: '{{DtSummary}}'
   alertTacticsColumnName: 
-  alertSeverityColumnName: 
```

Comment:
Same goes here, add severity to this removed column.

Reply:
That column hasn't been removed. It was updated. You can see what it was updated to below. It contains the value DtSentinelCategory 

<img width="809" alt="Screen Shot 2023-03-13 at 11 24 39 PM" src="https://user-images.githubusercontent.com/86425481/224914486-ef378e10-6033-4471-bfc7-008a7fe43b0a.png">

Reply:
I can still see it empty
![image](https://user-images.githubusercontent.com/10944026/230337661-63ac378f-f135-4337-b0d9-51aea729e4af.png)